### ISSUE TITLE
[Feature Request] Support url resolution in mermaid diagrams

### ISSUE THREAD

**Issue by sacrejohn** [dotnet/docfx#8772]
I would like to be able to add links in my mermaid diagrams and have these links resolved by the docfx engine

Example in the following diagram, I would like the 2 urls to be resolved automatically:
 
````
```mermaid
flowchart
  
    Component
    Page
  
    click Component "xref:UId"
    click Page "./page.md"
```
````

**Comment by Malte-Flammild** [comment]
I would also like the support of this through the DocFx engine.
@sacrejohn Did you find a workaround for this?

**Comment by sacrejohn** [comment]
While waiting for the full integration of the links, I’ve added relative URLs as follows:

    ```mermaid
    ---
    title: Organization
    ---
    flowchart BT
      
       subgraph Operational["Operational"]
          Support[Support]
          Administrator[Administrator]
       end    
        click Support "./Support/index.html"
        click Administrator "./Administrator/index.html"
    ```

This keeps the navigation functional for now. If you have any suggestions to improve this approach, I’m all ears. 😊

**Comment by Malte-Flammild** [comment]
I don't think this would solve my problems sadly. I'm looking to xref specific methods from a quite big code library. When DocFx is launched it generates xrefmap.yml and fills out xrefs in markdowns with hyper links. But just not in the mermaid diagrams.
Maybe a workaround would be to build two DocFx, one for code library, and the other for Mermaid, then a converter could swap the xrefs in mermaid with urls from xrefmap.yml. Only problems is having two instances of DocFx running referencing each other.
